packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/BlocksToolbar.tsx

```diff
@@ -26,8 +26,10 @@ import { insertLink } from './utils/links';
 import { type Block, getEntries, getKeys } from './utils/types';
 
 const ToolbarWrapper = styled<FlexComponent>(Flex)`
+  width: 100%;
```

Comment:
can you pass width="100%" as a prop on the component instead? we try to use the DS components props as much as possible for simple things

Reply:
@remidej does it mean doing this?

![CleanShot 2024-12-13 at 14 16 02@2x](https://github.com/user-attachments/assets/a42d3b13-4b0a-4695-a5e2-6f38e46685f7)